### DISCUSSION TITLE
Adding Sentry support

### DISCUSSION THREAD

**Discussion by iborysenko** [roadrunner-server/roadrunner#1475] [Ideas]
Hi @rustatian do you have any plans to implement RR sentry plugin ? 
RR already have NewRelic and Otel support, is it possible to create the same plugin for sentry? 
I'm using [ sentry distributed-tracing](https://docs.sentry.io/product/sentry-basics/tracing/distributed-tracing/) in my project, and it will be good to implement support this feature in the RR [transformation](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/sentryexporter/docs/transformation.md)

also log RR errors to sentry will be useful (for example rpc_call errors )

**Comment by rustatian** [comment]
Hey @iborysenko 👋🏻 

@butschster I think the SF has good Sentry support, right?

**Reply by iborysenko** [reply]
Yes, you  can use `sentry-symfony` bundle for enabling it, and it wrapped by default dbal connection & cache 
```yaml 
sentry:
  tracing:
    enabled: true
    dbal:
      enabled: true
    cache:
      enabled: false

```
and you can easily wrap another parts of your application like external http-calls,  jobs (and view all needed data)

and traces looks like <img width="1520" alt="Screenshot 2023-02-23 at 22 09 01" src="https://user-images.githubusercontent.com/17834238/221018749-2e736045-7b4f-40f4-8983-30af41538640.png">

And 
<img width="1182" alt="Screenshot 2023-02-23 at 22 10 23" src="https://user-images.githubusercontent.com/17834238/221019196-6455ec29-0bcd-434b-9109-d8e04e57c364.png">

the main benefit for my team, is because all errors and traces connected in one place, across all services (Frontend apps, NodeJs apps, BE http and  jobs services) 
We are using it 1+ year and it works stable with minimal impact

**Reply by iborysenko** [reply]
Of course I can live without distributed tracing support from RR, 
but I need to track RR errors, because my app can't track any errors outside the PHP worker

Like pipeline connection issues, my app will successfully use RR rpc-calls and do not know about possible issues

**Reply by rustatian** [reply]
NewRelic support was removed in favor of the OpenTelemetry plugin (OTEL). NewRelic supports the OTEL format and might be used without any problems.

Sentry supports the OTEL format as well; adding it would be an easy task. You need to create a feature request ticket; it'll be planned for the `2023.2.0` release milestone (since the 2023.1.0 window is closed). Minor version means quartal. 2 - means second quartal (end of June).
